doc/compiler/aot/InlinedMethods.md

```diff
@@ -0,0 +1,259 @@
+<!--
+Copyright (c) 2020, 2020 IBM Corp. and others
+
+This program and the accompanying materials are made available under
+the terms of the Eclipse Public License 2.0 which accompanies this
+distribution and is available at https://www.eclipse.org/legal/epl-2.0/
+or the Apache License, Version 2.0 which accompanies this distribution and
+is available at https://www.apache.org/licenses/LICENSE-2.0.
+
+This Source Code may also be made available under the following
+Secondary Licenses when the conditions for such availability set
+forth in the Eclipse Public License, v. 2.0 are satisfied: GNU
+General Public License, version 2 with the GNU Classpath
+Exception [1] and GNU General Public License, version 2 with the
+OpenJDK Assembly Exception [2].
+
+[1] https://www.gnu.org/software/classpath/license.html
+[2] http://openjdk.java.net/legal/assembly-exception.html
+
+SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0 WITH Classpath-exception-2.0 OR LicenseRef-GPL-2.0 WITH Assembly-exception
+-->
+
+# Overview
+
+AOT compilations support inlining of methods.
+However, ensuring validation and relocation is a non-trivial process with
+many subtleties. The relocation records related to the inlined methods are 
+different from the other records in that they do many things - they perform 
+validation, relocation of the inlined table entry in the J9JITExceptionTable
+metadata structure, as well as relocation of guards. 
+
+This document will go through how an inlined site is validated, as well what 
+what all is necessary as part of the relocation. Note, inlined method 
+relocations still follow the high level process described by the
+[Intro to AOT blog series](https://blog.openj9.org/2018/10/10/intro-to-ahead-of-time-compilation/),
```

Comment:
The link name should be the same as the actual article name, i.e. "Intro to Ahead Of Time Compilation". This helps search engines.